metafix/src/main/java/org/metafacture/metafix/Metafix.java

```diff
@@ -148,26 +150,43 @@ public void literal(final String name, final String value) {
     }
 
     public String resolvePath(final String path) {
-        final Path basePath;
+        String resolvedPath = path;
 
-        if (path.startsWith(".")) {
-            if (fixFile != null) {
-                basePath = getPath(fixFile).getParent();
+        if (isValidUrl(path)) {
```

Comment:
Why not assign `resolvedPath` here?

Reply:
> That demands the use of `path` to not have an initialized variable.

Can you elaborate? There is no change w.r.t. the `path` variable.

Reply:
I cannot assign `final String resolvedPath = path` to use `resolvedPath` in `if (isValidUrl(path)) {`, right? Maybe I just don't understand what you meant here.